.github/workflows/script/sw_build_and_push_gem.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+GEM_NAME="$1"
+
+case $GEM_NAME in
+  *metrics-sdk)
+    echo "metrics-sdk"
+    GEM_PATH="metrics_sdk/"
+    ;;
+  *metrics-api)
+    echo "metrics-api"
+    GEM_PATH="metrics_api/"
+    ;;
+  *exporter-otlp)
+    echo "exporter-otlp"
+    GEM_PATH="exporter/otlp/"
+    ;;
+  *)
+    echo "Unknown file type"
```

Comment:
I know this will `exit 1` with the next line at L24, but for clarity could this echo a message like

```suggestion
    echo "Unknown gem name $GEM_NAME. Aborting publish"
```

then exit immediately?

Reply:
I'd agree, I have changed to exit 1 immediately